auter.yumdnfModule

```diff
@@ -58,7 +58,7 @@ function apply_updates() {
     RC=0
     if [[ $(ls -1 ${DOWNLOADDIR}/${CONFIGSET}/*.rpm | wc -l) -gt 0 ]]; then
       RPMS="${DOWNLOADDIR}/${CONFIGSET}/*.rpm"
-      RC=100
+      RC=$("${PACKAGE_MANAGER}" check-update $(find "${DOWNLOADDIR}"/"${CONFIGSET}"/*.rpm -printf "%f\n" | sed 's/.rpm$//g') &>/dev/null;echo $?)
```

Comment:
This is pretty obtuse:
- i think we need a comment to explain what's happening
- lets put the handling of $? on another line to make it more readable
- you might be able to use this to make it a bit cleaner:
```
rpm -qp ${RPMS}
```